.devcontainer/chefs_local/local.json.sample

```diff
@@ -55,13 +55,25 @@
         "public": {
           "limitApiKey": "120",
           "limitFrontend": "500",
-          "windowMs": "60000",
+          "windowMs": "60000"
```

Comment:
I introduced this when hand-formatting the file. I'll create a task to rename the file to `local.sample.json` so that Prettier can do its thing.

Reply:
fixed